linux/build-appimage-daily.sh

```diff
@@ -93,6 +94,19 @@ export LD_LIBRARY_PATH=/app/usr/lib/
 # Build AppImage
 ./squashfs-root/AppRun ${DESKTOP_FILE} -appimage
 
+# Workaround issue #103
+rm -rf ./squashfs-root
+wget  -O appimagetool.AppImage --ca-directory=/etc/ssl/certs -c "https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage"
+chmod a+x appimagetool.AppImage
+./appimagetool.AppImage --appimage-extract
```

Comment:
Thanks for putting up this PR!
Is there a reason you extract `appimagetool` here?
Why not just run `./appimagetool-x86_64.AppImage -n ./squashfs-root "$APPIMAGE"`?

Reply:
As far as i understood, we need to extract it to delete the specific files that are in conflict from linuxdeployqt (see [this issue here](https://github.com/probonopd/linuxdeployqt/issues/544#issuecomment-1205099163))

Reply:
This is not what I wrote. The actual tool to re-package the modified Nextcloud AppImage is called `appimagetool` and is itself another AppImage. It's used to package app folders as AppImages. 
We can run `appimagetool` on the extracted Nextcloud folder directly, like I said in my previous comment.
I was just curious why Frederic extracts `appimagetool` before using it.

Reply:
good catch, thanks for the explanation, makes sense to leave the tool appImage packaged then as we only use it and dont modify it.

Reply:
I do the extraction otherwise you need to make FUSE works inside the container which is not possible (at least no easily).
I found this information here:
https://github.com/AppImage/AppImageKit/issues/405#issuecomment-305989062

The linuxdeployqt AppImage contains appimagetool. Maybe we can use it instead of downloading the appimagetool AppImage...

Reply:
>I do the extraction otherwise you need to make FUSE works inside the container which is not possible (at least no easily).

Ah, I see, I didn't know that, thanks!